pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -50,13 +50,7 @@ type HealthChecker struct {
 	interval   time.Duration
 }
 
-// NewHealthChecker creates a new HealthChecker object
-func NewHealthChecker(monCluster *Cluster) *HealthChecker {
-	h := &HealthChecker{
-		monCluster: monCluster,
-		interval:   HealthCheckInterval,
-	}
-
+func updateMonTimeout(monCluster *Cluster) {
 	monCRDTimeoutSetting := monCluster.spec.HealthCheck.DaemonHealth.Monitor.Timeout
```

Comment:
Are you seeing that `monCluster.spec` is updated when the cluster CR is updated? I'm not sure if it would be.

Reply:
Yes, they are being updated by cluster CR updation.